website/src/components/MiniBlog/index.tsx

```diff
@@ -111,7 +111,7 @@ const MiniBlog: React.FC = () => {
   ).length;
 
   const filteredData = (jsonMdData || []).filter((tabs: TabProps) => {
-    if (value !== "all") {
+    if (value !== t("blog.all").toLowerCase()) {
```

Comment:
taking this value from translation is not a good idea

Reply:
Can you please brief me about why we need to use a string here instead of translation?

Reply:
The idea of adding translation is for adding it in different languages, in this case it should come from a constant.